src/resources.js

```diff
@@ -1307,6 +1319,37 @@ export class Resource {
       this.state_ = ResourceState_.NOT_LAID_OUT;
     }
   }
+
+  /**
+   * Only allowed in dev mode when runtime is turned off. Performs all steps
```

Comment:
We have DEV mode fully compiled into the main binary and the switch happens in the runtime. So I don't think this can get DCE'd. I tried to move as much as possible into the Phantom script itself though.